src/public/CowSwap.json

```
@@ -564,6 +564,14 @@
       "decimals": "18",
       "chainId": 1,
       "logoURI": "https://raw.githubusercontent.com/cowprotocol/token-lists/main/src/public/images/1/0xfcf8eda095e37a41e002e266daad7efc1579bc0a/logo.png"
+    },
+    {
+      "address": "0x19062190b1925b5b6689d7073fdfc8c2976ef8cb",
+      "symbol": "BZZ",
+      "name": "Swarm",
+      "decimals": "16",
```

Fixing decimals (already fixed for new workflows)

```suggestion
      "decimals": 16,
```